### ISSUE TITLE
Reproducibility

### ISSUE THREAD

**Issue by hchoi256** [zaqai/LGCount#2]
_______________________________________________________________________________________________________________________________________________________________________________
       Test metric             DataLoader 0
_______________________________________________________________________________________________________________________________________________________________________________
        test_mae            17.955987152749234
        test_rmse           61.843953708005756
_______________________________________________________________________________________________________________________________________________________________________________
====Metric on test set====
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [1]
Testing DataLoader 0: 100%|________________________________________________________________________________________________________________| 1190/1190 [02:15<00:00,  8.79it/s]
_______________________________________________________________________________________________________________________________________________________________________________
       Test metric             DataLoader 0
_______________________________________________________________________________________________________________________________________________________________________________
        test_mae             17.32410002956871
        test_rmse            101.4000965532707
_______________________________________________________________________________________________________________________________________________________________________________

Thanks for your work, and I find it very interesting. So, I try to reproduce the results.
Following the instruction on README.md, I reproduced LGCount and got the results above different from the what's noted in the paper and the README. Did I miss something?

**Comment by Liwen-Xiao** [comment]
Hello, may I ask whether you directly used the checkpoint provided by the authors to test on FSC147 and obtained these results?

**Comment by hchoi256** [comment]
> Hello, may I ask whether you directly used the checkpoint provided by the authors to test on FSC147 and obtained these results?

```
// run.py
        ck = torch.load('./ckpt/epoch=149-avg_fine_accuracy_pred=0.71.ckpt', map_location='cpu')

        # ck=torch.load(self.args.align_path)
        # print(self.args.align_path)
        selected_weights = {}
        # for i in ck['state_dict']:
        #     selected_weights[i[6:]]=ck['state_dict'][i]
        # self.model_align.load_state_dict(selected_weights,strict=False)
        params = ['img_encoder.visual_prompt',
                  'img_encoder.vpt_norm.weight',
                  'img_encoder.vpt_norm.bias',
                  'img_encoder.vpt_proj.weight',
                  'img_encoder.vpt_proj.bias',
                  'text_encoder.learnable_context']
        for k in params:
            selected_weights[k] = ck['state_dict']['model.' + k]
            # self.model_align.load_state_dict({k:ck['state_dict']['model.'+k]},strict=False)
        self.model_align.load_state_dict(selected_weights, strict=False)
        self.model_align.requires_grad_(False)
```

Yes, I used the checkpoint `epoch=149-avg_fine_accuracy_pred=0.71.ckpt` following the instruction in README.md.
